{{cookiecutter.repo_name}}/manage.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
```

Comment:
It seems that this file isn't used, if so please remove it.

Unless we have to use to for `runserver`, which is AFAIK not the case.

Reply:
As said before, I'm moving back to this to fix the issues mentioned.